src/core/models.py

```diff
@@ -55,6 +55,7 @@ def profile_images_upload_path(instance, filename):
     ('Ms', 'Ms'),
     ('Mrs', 'Mrs'),
     ('Mr', 'Mr'),
+    ('Mx', 'Mx'),
```

Comment:
We are going to need a migration  on `core` for this change I think.

Reply:
Ooh, yes. I think I can just add that retrospectively to the initial migration, as it doesn't have any actual effect on the database schema (i.e. no `ALTER` needs to be emitted).

Is that okay, or a bit unintuitive compared to a fresh migration?

Reply:
yep, that is fine by me